libs/SalesforceHybridSDK/SalesforceHybridSDKTestApp/Classes/AppDelegate.m

```diff
@@ -153,9 +157,29 @@ - (void)userAccountManager:(SFUserAccountManager *)userAccountManager
 }
 
 #pragma mark - Private methods
+- (SFUserAccount *) createInterimTestUser:(SFSDKTestCredentialsData *) credsData  {
```

Comment:
There are some competing paradigms and overlapping duties here with `[TestSetupUtils populateAuthCredentialsFromConfigFileForClass:]`.  I'll make some recommendations there, and then a lot of this may be redundant and go away.  Stay tuned!